dali/operators/signal/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if (BUILD_FFTS)
+  add_subdirectory(fft)
+endif()
```

Comment:
This suggests, that ffts is the only way to calculate FFT in DALI. Aren't we using specific lib for powers of 2?

Reply:
for now it is the only way. Once we start supporting genFFT we can then enable FFT implementation with BUILD_FFTS=OFF